packages/@aws-cdk/aws-scheduler-targets-alpha/lib/event-bridge-put-events.ts

```diff
@@ -0,0 +1,91 @@
+import { IScheduleTarget, ISchedule, ScheduleTargetInput, ScheduleTargetConfig } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import * as events from 'aws-cdk-lib/aws-events';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * An entry to be sent to EventBridge
+ *
+ * @see https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEventsRequestEntry.html
+ */
+export interface EventBridgePutEventsEntry {
+  /**
+   * The event body
+   *
+   * Can either be provided as an object or as a JSON-serialized string
+   * @example
```

Comment:
with `@example`, eventally `jsii-rosetta` (a tool we have that aggregates all examples that show up in our documentation) will want this to compile. Unless you want to figure out how to get this to compile, I suggest the following:

```suggestion
   * For example,
   *
   * ```
   * // your uncompiled example code here
   * ```
```

Reply:
The build is green as far as I know. Or which build (or command) do you refer to?

Reply:
It does compile :). Nevermind.